pkg/render/node.go

```diff
@@ -1454,6 +1455,7 @@ func (c *nodeComponent) nodeEnvVars() []corev1.EnvVar {
 				corev1.EnvVar{Name: "FELIX_PROMETHEUSREPORTERCERTFILE", Value: c.cfg.PrometheusServerTLS.VolumeMountCertificateFilePath()},
 				corev1.EnvVar{Name: "FELIX_PROMETHEUSREPORTERKEYFILE", Value: c.cfg.PrometheusServerTLS.VolumeMountKeyFilePath()},
 				corev1.EnvVar{Name: "FELIX_PROMETHEUSREPORTERCAFILE", Value: c.cfg.TLS.TrustedBundle.MountPath()},
+				corev1.EnvVar{Name: "FELIX_PROMETHEUSREPORTERFIPSMODEENABLED", Value: fmt.Sprintf("%v", c.cfg.Installation.FIPSMode == operatorv1.FIPSModeEnabled)},
```

Comment:
I thought we said we're bundling all of the FELIX_XFIPSMODEENABLED into a single FIPS_MODE_ENABLED var? 

>  fips mode is a cluster wide setting

There's not really a use case for turning FIPS mode on per-connection to Felix, it's either on for all of them or off for all of them.